defaults/main.yml

```diff
@@ -14,6 +14,7 @@ postgresql_default_auth_method: "trust"
 # The user/group that will run postgresql process or service
 postgresql_service_user: "{{ postgresql_admin_user }}"
 postgresql_service_group: "{{ postgresql_admin_user }}"
+postgresql_service_enabled: false
```

Comment:
Why postgresql service disabled by default? Service should be enabled by default. And in README you wrote enabled default value.